web-registry/src/pages/Home/Home.loader.ts

```diff
@@ -0,0 +1,74 @@
+import { QueryClient } from '@tanstack/react-query';
+
+import { getEcocreditQueryClient } from 'lib/clients/ecocreditQueryClient';
+import { getMarketplaceQueryClient } from 'lib/clients/marketplaceQueryClient';
+import { getSimplePriceQuery } from 'lib/queries/react-query/coingecko/simplePrice/simplePriceQuery';
+import { getProjectsQuery } from 'lib/queries/react-query/ecocredit/getProjectsQuery/getProjectsQuery';
+import { getAllowedDenomQuery } from 'lib/queries/react-query/marketplace/getAllowedDenomQuery/getAllowedDenomQuery';
+import { getSellOrdersExtentedQuery } from 'lib/queries/react-query/marketplace/getSellOrdersExtentedQuery/getSellOrdersExtentedQuery';
+import { getAllCreditClassesQuery } from 'lib/queries/react-query/sanity/getAllCreditClassesQuery/getAllCreditClassesQuery';
+import { getAllHomePageQuery } from 'lib/queries/react-query/sanity/getAllHomePageQuery/getAllHomePageQuery';
+import { getFromCacheOrFetch } from 'lib/queries/react-query/utils/getFromCacheOrFetch';
+
+import { client as sanityClient } from '../../sanity';
+
+type LoaderType = {
+  queryClient: QueryClient;
+};
+
+export const homeLoader =
```

Comment:
Note that loaders are completely optional, a page should work even without them.
There's just a way to trigger queries early on.
Also queries with `await` block page render while those without do not.